MonoGame.Framework/Utilities/OS.cs

```diff
@@ -0,0 +1,46 @@
+﻿// MonoGame - Copyright (C) The MonoGame Team
+// This file is subject to the terms and conditions defined in
+// file 'LICENSE.txt', which is part of this source code package.
+
+namespace MonoGame.Utilities
+{
+    /// <summary>
+    /// Defines the operating system.
+    /// </summary>
+    public enum OS
```

Comment:
If some Linux user has a need to create a Library folder your code will fail, I don't like adding code that has a chance of failure with something so simple unless I have to.